frontend/src/views/web/challenge/challenge-page.html

```diff
@@ -42,6 +42,23 @@
                                 </a>
                             </span>
                         </span>
+
+                        <br>
+                        <span>
+                            <a class="pointer" ng-click="challenge.challengeDateDialog($event)">
+                                <strong class="text-light-black">Starts on:</strong>
+                                {{ challenge.page.start_date | date:'medium' }}
```

Comment:
@takitsuse Please also add `time-zone` -> `{{ challenge.page.start_date | date:'medium' }} {{challenge.page.start_zone}}` here and also for `End Date`

Reply:
We see `{{ challenge.start_zone }}` on the challenge list, but don't on the hosted challenge list.
Since this file is about hosted challenge editation, I'd say we don't need `{{ challenge.start_zone}}` here.